Web.R.tex

```diff
@@ -714,25 +718,26 @@ \section{Other solutions}\label{sec:browserprojects}
 \begin{itemize}
 	\item \textbf{SAFE Network}\footnote{SAFE Network: \url{https://safenetwork.org/}}: a project developed by MaidSafe\footnote{MaidSafe: \url{https://maidsafe.net/}} that aims to reuse available hard disk space of computers to provide a distributed storage system, which can be accessed and explored with the SAFE browser; at the time of writing, this project is only available on an Alpha test network;
 	\item \textbf{Beaker Browser}\footnote{Beaker Browser: \url{https://beakerbrowser.com/}}: based on the dataset synchronization protocol known as Dat, Beaker Browser allows for creation and visualization of distributed websites, although it fails to provide or reuse a distributed naming system, relying instead on HTTP and DNS to resolve names;
-	\item \textbf{Freenet}\footnote{Freenet: \url{https://freenetproject.org/}}: a very mature project, with its initial release dating back to the year 2000, it's a peer-to-peer platform focused on anonymity and censorship resistance; Freenet allows browsing websites, known as \emph{freesites}, but the naming scheme is based on nicknames, which are simply labels defined and stored by single users and not shared globally, thus not defying Zooko's Triangle.
+	\item \textbf{Freenet}\footnote{Freenet: \url{https://freenetproject.org/}}: a very mature project, with its initial release dating back to the year 2000, it's a peer-to-peer platform focused on anonymity and censorship resistance; Freenet allows browsing websites, known as \emph{freesites}, but the naming scheme is based on nicknames, which are simply labels defined and stored by single users and not shared globally. % , thus not defying Zooko's Triangle.
+      %todo(leandro) - I've cut the small part above because it may be mistaken as if Zooko's Triangle has not been refuted (and you mention that Namecoin does).
```

Comment:
Freenet's non-refusal of Zooko's Triangle is the only reason for which Freenet was not included in the Thesis. Are we sure we want to cut it?

Reply:
I see, no problem. It's only the word "refute" that might lead to a different interpretation.
Also, I'd mention the specific property that it does not satisfy. Something like:
", thus not satisfying property X from Zooko's Triangle."

Reply:
I will do that.
Since the violated property is securty, I will also expand that definition a bit, by changing it from:
`an attacker cannot fake name resolutions`
to:
`name resolutions will yield the same result for everyone, and an attacker cannot fake name resolutions`

Reply:
I think this last phrase sounds a bit repetitive and might be slightly wrong, in the sense that "the same result for everyone" is too vague (e.g., what if you see the new value but i'm still seeing the old one? that isn't necessarily "wrong", depending on the consistency guarantees).

Reply:
Actually, I'm wrong, Freenet nicknames are secure but not decentralized (obviously). I will leave the definitions as they are.